scripts/build-espresso.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const { logger, fs } = require('@appium/support');
+const { ServerBuilder } = require('../build/lib/server-builder.js');
+
+const LOG = new logger.getLogger('EspressoBuild');
+
+const ROOT_DIR = path.resolve(__dirname, '..');
+const ESPRESSO_SERVER_ROOT = path.join(ROOT_DIR, 'espresso-server');
+
+const ESPRESSO_SERVER_BUILD = path.join(ESPRESSO_SERVER_ROOT, 'app', 'build');
+
+async function buildEspressoServer () {
+  console.log(`Deleting the build directory ${ESPRESSO_SERVER_BUILD}`); // eslint-disable-line no-console
+
+  const opts = {
+    serverPath: ESPRESSO_SERVER_ROOT,
+    showGradleLog: process.env.SHOW_GRADLE_LOG ? process.env.SHOW_GRADLE_LOG : false
```

Comment:
not sure if we need to check the presence of env variables or also explicitly check if their values are not false/0

Reply:
Changed to enable it only when the given env var was 1 or true. Other cases will be false as same as the default behavior.